src/main/drivers/dshot_bitbang_decode.c

```diff
@@ -142,7 +140,7 @@ uint32_t decode_bb_bitband( uint16_t buffer[], uint32_t count, uint32_t bit)
         return DSHOT_TELEMETRY_NOEDGE;
     }
 
-    int remaining = MIN(count - (p - b), (unsigned int)MAX_VALID_BBSAMPLES);
+    const uint32_t remaining = MIN(count - startMargin, (unsigned int)MAX_VALID_BBSAMPLES);
```

Comment:
Personally, I'd use `int` here.
- 32 bits will only confuse reader (values are actually small)
- Can you prove that value is always positive? I only looked on surrounding code and I'm not sure. Rest of this code will handle negative `remaining`, so better be on safe side